spec/requests/api/v1/lotteries_spec.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'api/v1/lotteries', type: :request do
+  describe 'POST /api/v1/lotteries' do
+    context 'when the Authorization header is valid' do
+      let(:headers) { { Authorization: 'Token 1234' } }
+
+      context 'when there are people eligible for lottering' do
+        let(:people) do
+          [
+            { 'name' => 'Isaac Newton', 'cpf' => '000.001.003-05', 'birthday' => '1943-01-04' },
+            { 'name' => 'Leonhard Euler', 'cpf' => '271.828.182-84', 'birthday' => '1707-04-15' },
+            { 'name' => 'Felipe', 'cpf' => '000.000.000', 'birthday' => '1900-01-01' }
+          ]
+        end
+
+        before do
+          Person.create!(name: 'Isaac Newton', cpf: '000.001.003-05', birthday: '1943-01-04')
+          Person.create!(name: 'Leonhard Euler', cpf: '271.828.182-84', birthday: '1707-04-15')
+          Person.create!(name: 'Felipe', cpf: '000.000.000', birthday: '1900-01-01')
+        end
+
+        it 'creates a new lottery' do
+          expect do
+            post api_v1_lotteries_path, headers:
+          end.to change { Lottery.count }.by(1)
```

Comment:
só estilo, eu prefiro {} quando tenho que fazer {}.to 💅

Reply:
Eu também prefiro. Mas quando eu executo o rubocop ele muda para "do ... end" devo mudar isso ou deixar o que o rubocop colocou?